composer/src/main/kotlin/com/gojuno/composer/Apk.kt

```diff
@@ -66,4 +65,5 @@ fun parseTestRunner(testApkPath: String): TestRunner =
                 .toBlocking()
                 .value()
 
-fun parseTests(testApkPath: String) = DexParser.findTestMethods(testApkPath)
\ No newline at end of file
+fun parseTests(testApkPath: String) : List<TestMethod> =
```

Comment:
We can add an integration test (we already have some that parse runner from test apk, you can  add couple tests there)

Not that I don't trust this library, but it can help catch future regressions!